src/rgw/rgw_crypt.cc

```diff
@@ -16,6 +16,11 @@
 #include "include/str_map.h"
 #include "crypto/crypto_accel.h"
 #include "crypto/crypto_plugin.h"
+
+#ifdef USE_OPENSSL
+# include <openssl/evp.h>
+#endif // USE_OPENSSL
+
```

Comment:
#else/#elif?

Reply:
Fixed by keeping the `#else` clause we had for NSS.